### ISSUE TITLE
validate whether the sum of all account balances is smaller or equal to the channel balance held in the underlying LND node

### ISSUE THREAD

**Issue by AndySchroder** [lightninglabs/lightning-terminal#495]
**Is your feature request related to a problem? Please describe.**
In the documentation, it says "The accounts feature does not validate whether the sum of all account balances is smaller or equal to the channel balance held in the underlying LND node."

**Describe the solution you'd like**
It seems like every time a new account is created, a check should exist to make sure that the amount being allocated to the account is not greater than the total outbound capacity of the node minus the current sum of the amounts in all the other accounts.

**Additional context**
It's possible this statement that I've quoted is being misinterpreted? Is there some limitation related to channel capacity inbound vs outbound that makes this complicated and I don't realize it?

**Comment by ellemouton** [comment]
@AndySchroder , I think the complexity comes in later on. The LND node owner always has the ability to spend all funds on the node. So even if at the time of opening an account, the outbound capacity is enough to support the account, the node owner could then later on spend all the funds of the node and then the account balance is no longer backed. 

The accounts feature is an RPC interceptor implemented on Lit. So LND itself is un-aware of accounts

**Comment by AndySchroder** [comment]
So, would it be good policy to do the following?

1. A node operator has a "main" account using the new accounts feature in Lit, then use that for normal administrative activity
2. For any new accounts, if the account is created with a non-zero balance, subtract that automatically from the "main" account, giving an error message if there is insufficient balance.
3. Closing an account returns the remaining balance back to the "main" account.
4. Don't use the LND grpc directly anymore except in emergency administrative circumstances.

**Comment by ellemouton** [comment]
Yes that could technically work but still a danger of the main node owner using the LND grpc directly (even if accidentally) which would break the assumptions of the accounts managing system. So still a tricky problem I think.
